packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5553,6 +5553,62 @@ class ExcludeSemantics extends SingleChildRenderObjectWidget {
   }
 }
 
+/// A widget that annotates the child semantics with an index.
+///
+/// Certain widgets like the [ListView] will automatically provide a child
+/// index for building semantics. A user may wish to manually provide semanitc
+/// indexes if not all child of the scrollable contribute semantics.
+///
+/// ## Sample code
+///
+/// The example below handles spacers in a scrollable that don't contribute
+/// semantics. The automatic indexes would give the spaces a semantic index,
+/// causing scroll announcements to erroneously state that there are four items
+/// visible.
+///
+/// ```dart
```

Comment:
@gspencergoog do you know why this sample code section doesn't work?.

Reply:
"Doesn't work" in what sense?

Reply:
https://github.com/flutter/flutter/pull/21764/checks?check_run_id=21624772 The sample code check thinks it is a class/method declaration

Reply:
It looks "normal" to me.  You might check at the top of the file for a `// Examples can assume:` block that breaks this code, maybe.